src/Kunstmaan/MediaBundle/Entity/CroppableMediaLink.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Kunstmaan\AdminBundle\Entity\AbstractEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="kuma_croppable_media_link")
+ * @ORM\HasLifecycleCallbacks
+ */
+class CroppableMediaLink extends AbstractEntity
+{
+    /**
+     * @ORM\ManyToOne(targetEntity="Kunstmaan\MediaBundle\Entity\Media", cascade={"persist"})
+     * @ORM\JoinColumn(name="media_id", referencedColumnName="id")
+     * @Assert\NotNull()
+     */
+    private $media;
+
+    /**
+     * @ORM\Column(name="runtime_config", type="text", nullable=true)
```

Comment:
Maybe it's better to use the doctrine json type directly here and let doctrine handle the encode/decode? Wdyt?

This will also allow the database to optimize the actual data storage. For example newer mysql servers and postgres have extended json support in their engines. Also our code would be sure to always get an array of data

Reply:
Its because the frontend writes the array already json encoded in the hidden textfield in the form. otherwise im going to have to decode it in the form type just to have doctrine encode it again when saving